### PR TITLE
Fix FIM test_large_changes module

### PR DESCRIPTION
**Pull request by Deblintrake09**
|Related issue|
|-------------|
| #3860     |

## Description

This PR fixes and re-enables the `test_fim/test_files/test_report_changes/test_large_changes.py` module.

### Updated

- Fixed `test_fim/test_files/test_report_changes/test_large_changes.py`

## Testing performed

| Tester             | Test path | Jenkins | Local  | OS | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|----------------------|
| @Deblintrake09  (Developer)  |  test_fim         | [:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39335)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39340)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39339)  | [:green_circle::green_circle::green_circle:](https://github.com/wazuh/wazuh-qa/files/11605854/3948-Manager.zip)  | Manager        |  https://github.com/wazuh/wazuh-qa/commit/9ee87320b86be36443f47c119e745023c38b6008       | Nothing to highlight |
| @Deblintrake09  (Developer)  |  test_fim         | [:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39338)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39341)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39342)  |  | Centos Agent        |  https://github.com/wazuh/wazuh-qa/commit/9ee87320b86be36443f47c119e745023c38b6008       | Nothing to highlight |
| @Deblintrake09  (Developer)  |  test_fim         | [:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39634)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39637)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39638)  | [:green_circle::green_circle::green_circle:](https://github.com/wazuh/wazuh-qa/files/11604215/3948-WinAgent.zip)   | Windows Agent        |  https://github.com/wazuh/wazuh-qa/commit/9ee87320b86be36443f47c119e745023c38b6008       | Nothing to highlight |
| @Deblintrake09  (Developer)  |  test_fim         | [:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39633)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39635)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/39636)  | | Solaris & macOS        |  https://github.com/wazuh/wazuh-qa/commit/9ee87320b86be36443f47c119e745023c38b6008       | Nothing to highlight |

### CODE REVIEW

**Review by damarisg — APPROVED** [review]

File: tests/integration/test_fim/test_files/test_report_changes/data/test_cases/cases_large_changes.yaml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  configuration_parameters:
+    INTERVAL: 10000
+    REALTIME: 'no'
+    WHODATA: 'yes'
+  metadata:
+    filename: regular_2
+    original_size: 200000
+    modified_size: 200000
+    has_more_changes: true
+    fim_mode: whodata
```
Comment (damarisg) [inline tests/integration/test_fim/test_files/test_report_changes/data/test_cases/cases_large_changes.yaml:77]: Please, update the description.

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py [hunk 2]
```diff
@@ -147,38 +116,35 @@ def test_large_changes(filename, folder, original_size, modified_size, get_confi
                  the test will verify that the generated FIM event contains in its 'content_changes' field the proper
                  value depending on the test case.
 
-    wazuh_min_version: 4.2.0
+    wazuh_min_version: 4.5.0
```
Comment (damarisg) [inline tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py:119]: The test was skipped for inconsistent, but it should work on 4.2. So, Why did you change the min version?

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py [hunk 3]
```diff
@@ -147,38 +116,35 @@ def test_large_changes(filename, folder, original_size, modified_size, get_confi
...
+            type: dict
+            brief: Test case data.
+        - test_folders:
+            type: dict
+            brief: List of folders to be created for monitoring.
+        - set_wazuh_configuration:
             type: fixture
-            brief: Get configurations from the module.
-        - configure_environment:
+            brief: Set ossec.conf configuration.
```
Comment (damarisg) [inline tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py:135]: Please, update the description without the specific name file.

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py [hunk 4]
```diff
@@ -147,38 +116,35 @@ def test_large_changes(filename, folder, original_size, modified_size, get_confi
...
+        - create_monitored_folders_module:
             type: fixture
-            brief: Configure a custom environment for testing.
-        - configure_local_internal_options_module:
+            brief: Create a given list of folders when the module starts. Delete the folders at the end of the module.
+        - configure_local_internal_options_function:
             type: fixture
-            brief: Configure the local internal options file.
-        - restart_syscheckd:
+            brief: Set local_internal_options.conf file.
```
Comment (damarisg) [inline tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py:141]: Please, update the description without the specific name file.

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py [hunk 5]
```diff
@@ -147,38 +116,35 @@ def test_large_changes(filename, folder, original_size, modified_size, get_confi
...
+        - restart_syscheck_function:
             type: fixture
-            brief: Clear the 'ossec.log' file and start a new monitor.
-        - wait_for_fim_start:
+            brief: restart syscheckd daemon, and truncate the ossec.log.
```
Comment (damarisg) [inline tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py:144]: Please, update the description without the specific name file.

**Review by damarisg — CHANGES_REQUESTED** [review]

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py (hunk 2 above)
Comment (Deblintrake09) [reply tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py:119]: This was changed because it was affected by the DBSync update and messages changed as of 4.5

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py (hunk 5 above)
Comment (Deblintrake09) [reply tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py]: Done in https://github.com/wazuh/wazuh-qa/pull/4219/commits/d664f239c801fb219b9c529f5320e36c9508961b

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py (hunk 4 above)
Comment (Deblintrake09) [reply tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py]: Done in https://github.com/wazuh/wazuh-qa/pull/4219/commits/d664f239c801fb219b9c529f5320e36c9508961b

File: tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py (hunk 3 above)
Comment (Deblintrake09) [reply tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py]: Done in https://github.com/wazuh/wazuh-qa/pull/4219/commits/d664f239c801fb219b9c529f5320e36c9508961b

File: tests/integration/test_fim/test_files/test_report_changes/data/test_cases/cases_large_changes.yaml [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  configuration_parameters:
+    INTERVAL: 10000
+    REALTIME: 'no'
+    WHODATA: 'yes'
+  metadata:
+    filename: regular_4
+    original_size: 200000
+    modified_size: 10
+    has_more_changes: true
+    fim_mode: whodata
```
[the remark this answers is not in the document]
Comment (Deblintrake09) [reply tests/integration/test_fim/test_files/test_report_changes/data/test_cases/cases_large_changes.yaml]: Done in https://github.com/wazuh/wazuh-qa/pull/4219/commits/d664f239c801fb219b9c529f5320e36c9508961b

File: tests/integration/test_fim/test_files/test_report_changes/data/test_cases/cases_large_changes.yaml (hunk 1 above)
Comment (Deblintrake09) [reply tests/integration/test_fim/test_files/test_report_changes/data/test_cases/cases_large_changes.yaml]: Done in https://github.com/wazuh/wazuh-qa/pull/4219/commits/d664f239c801fb219b9c529f5320e36c9508961b

**Review by damarisg — APPROVED** [review]
